pkg/agent/baker_test.go

```diff
@@ -1737,14 +1737,14 @@ var _ = Describe("getGPUDriverVersion", func() {
 		Expect(getGPUDriverVersion("standard_nc6")).To(Equal(datamodel.Nvidia470CudaDriverVersion))
 	})
 	It("should use 525 cuda with nc v3", func() {
-		Expect(getGPUDriverVersion("standard_nc6_v3")).To(Equal(datamodel.Nvidia525CudaDriverVersion))
+		Expect(getGPUDriverVersion("standard_nc6_v3")).To(Equal(datamodel.Nvidia535CudaDriverVersion))
 	})
 	It("should use 535 grid with nv v5", func() {
 		Expect(getGPUDriverVersion("standard_nv6ads_a10_v5")).To(Equal(datamodel.Nvidia535GridDriverVersion))
 		Expect(getGPUDriverVersion("Standard_nv36adms_A10_V5")).To(Equal(datamodel.Nvidia535GridDriverVersion))
 	})
-	It("should use 525 cuda with nv v1 (although we don't know if that works)", func() {
-		Expect(getGPUDriverVersion("standard_nv6")).To(Equal(datamodel.Nvidia525CudaDriverVersion))
+	It("should use 535 cuda with nv v1 (although we don't know if that works)", func() {
```

Comment:
"Although we don't know if that works" seems mans couldn't get GPU Quota

Reply:
That's a question to our ancestors

Reply:
Checked - those SKUs are retired now, will keep this test just for safety and will add a note about retirement.

Reply:
https://learn.microsoft.com/en-us/azure/virtual-machines/nv-series?source=recommendations